CheckItem.h

```diff
@@ -11,6 +11,8 @@ in the source distribution for its full text.
 
 #include "Object.h"
 
+#include <stdbool.h>
+
```

Comment:
This looks like they current code is non-standardly using C99 `bool` without include stdbool.h. This could probably be done as a separate PR just to shrink this PR a bit.